notes.md

```
@@ -16,3 +16,7 @@
 
 <!-- Change related post from slider in product details -->
 TODO: add cron to delete session carts after 24 hours (and their items)
+TODO: orders with invoice and details 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)